README.md

```diff
@@ -22,42 +22,36 @@ This repository contains the source code for the Heimdall's [Backend](https://gi
-| Multiple Report Output<br />(DISA Checklist XML, CAT, XCCDF-Results, and more) |                                                          |                              x                               |
-| Authenticated REST API                                       |                                                          |                              x                               |
-| InSpec Run 'Delta' View                                      |                                                          |                              x                               |
-| Multi-Report Tagging, Filtering and Delta View               |                                                          |                              x                               |
+| Additional Installation Requirements    |      |     Postgres Server |
+| Overview Dashboard & Counts | x | x |
+| Deep Dive View of Security Control Results and Metadata | x | x |
+| 800-53 Partition and TreeMap View     | x | x |
+| Comparison View      | x | x |
+| Advanced Data / Filters for Reports and Viewing     | x |  x |
+| Multiple Report Output<br />(DISA Checklist XML, CAT, XCCDF-Results, and more) | x | x |
+| View Multiple Guidance Formats (InSpec profile, Checklist, DISA & CIS XCCDF) | x | x |
+| Automatic Conversion of [Various Security Formats](https://saf-cli.mitre.org/) | x | x |
+| Authenticated REST API       |   | x |
+| Users & Roles & multi-team support    |   | x |
+| Authentication & Authorization        | Hosting Webserver | Hosting Webserver<br />LDAP<br />OAuth Support for:<br /> GitHub, GitLab, Google, and Okta. |
 
 ### Use Cases
 
-| [Heimdall-Lite](https://github.com/mitre/heimdall-lite/) | [Heimdall](https://github.com/mitre/heimdall/)           |
+| Heimdall (Heimdall-Lite) | Heimdall with Backend  |
 | :------------------------------------------------------: | :------------------------------------------------------: |
-|            Ship the App & Data via simple Email          |                 Multiple Teams Support                   |
-|            Minimal Footprint & Deployment Time           |               Timeline and Report History                |
-|                Local or disconnected Use                 |               Centralized Deployment Model               |
-|                 One-Time Quick Reviews                   |       Need to view the delta between one or more runs    |
-|                Decentralized Deployment                  |   Need to view subsets of the 800-53 control alignment   |
-|                    Minimal A&A Time                      | Need to produce more complex reports in multiple formats |
-
-
+| Just-in-Time Use | Multiple Teams |
+| Minimal Footprint & Deployment Time  | Timeline and Report History |
+| Local or disconnected Use | Centralized Deployment Model |
+| Minimal Authorization & Approval Time |  |
 
 ## Getting Started / Installation
```

Comment:
I believe that we need to streamline the instructions here, propose the following:
- List installation types in a TOC with references  (#) to the instruction, this provides the user a listed view of the types of installations.
- Keep the Heimdall Server name consistent with how e are using above Heimdall with Backend
- Not enough detail on how to install via k8s

Reply:
Added TOC. I agree about keeping the naming consistent which I need to update. I also understand the helm chart point, however I think it easier to keep up with documentation only in one place and the helm chart repo has its own README